curriculum/challenges/english/25-front-end-development/quiz-css-accessibility/66ed8fc1f45ce3ece4053ead.md

```diff
@@ -17,439 +17,434 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --distractors--
 
-Placeholder distractor 1
+display: none;
 
 ---
 
-Placeholder distractor 2
+visibility: hidden;
 
 ---
 
-Placeholder distractor 3
+opacity: 0;
 
 #### --answer--
 
-Placeholder answer
+All of the above
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the minimum contrast ratio for normal text according to WCAG?
 
 #### --distractors--
 
-Placeholder distractor 1
+2:1
 
 ---
 
-Placeholder distractor 2
+3:1
 
 ---
 
-Placeholder distractor 3
+5:1
 
 #### --answer--
 
-Placeholder answer
+4.5:1
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which tool can help inspect element styles in a web browser?
 
 #### --distractors--
 
-Placeholder distractor 1
+Adobe Photoshop
 
 ---
 
-Placeholder distractor 2
+Microsoft Word
 
 ---
 
-Placeholder distractor 3
+Notepad
 
 #### --answer--
 
-Placeholder answer
+Chrome DevTools
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which attribute is used to hide content from screen readers?
 
 #### --distractors--
 
-Placeholder distractor 1
+hidden
 
 ---
 
-Placeholder distractor 2
+display: none;
 
 ---
 
-Placeholder distractor 3
+visibility: hidden;
 
 #### --answer--
 
-Placeholder answer
+aria-hidden
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the ideal contrast ratio for large text?
 
 #### --distractors--
 
-Placeholder distractor 1
+2:1
 
 ---
 
-Placeholder distractor 2
+4.5:1
 
 ---
 
-Placeholder distractor 3
+3.5:1
 
 #### --answer--
 
-Placeholder answer
+3:1
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which online tool checks color contrast accessibility?
 
 #### --distractors--
 
-Placeholder distractor 1
+ColorZilla
 
 ---
 
-Placeholder distractor 2
+Canva
 
 ---
 
-Placeholder distractor 3
+Figma
 
 #### --answer--
 
-Placeholder answer
+Contrast Checker
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which CSS property hides an element from visual rendering?
 
 #### --distractors--
 
-Placeholder distractor 1
+visibility
 
 ---
 
-Placeholder distractor 2
+position
 
 ---
 
-Placeholder distractor 3
+float
 
 #### --answer--
 
-Placeholder answer
+display
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is a recommended contrast ratio for accessible text?
 
 #### --distractors--
 
-Placeholder distractor 1
+1.5:1
 
 ---
 
-Placeholder distractor 2
+7:1
 
 ---
 
-Placeholder distractor 3
+3:1
 
 #### --answer--
 
-Placeholder answer
+4.5:1
 
 ### --question--
 
 #### --text--
-
-Placeholder question
+What does the 'Inspect Element' feature do?
 
 #### --distractors--
 
-Placeholder distractor 1
+Edits text directly
 
 ---
 
-Placeholder distractor 2
+Downloads files
 
 ---
 
-Placeholder distractor 3
+Modifies images
 
 #### --answer--
 
-Placeholder answer
+Displays CSS and HTML
 
 ### --question--
 
 #### --text--
-
-Placeholder question
+Which method keeps content in the document but hidden visually?
```

Comment:
`visibility: hidden` removes the element from the accessibility tree and thus hides it from people using assistive technology, such as screen readers. So while technically the element might still show in the DOM inspector, it really isn't there for people using assistive technology. Unfortunately, there is no single property that can properly "visually hide" an element at this time.